src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -273,10 +273,20 @@ public void deleteTicket(long id) {
         complete(submit(req("DELETE", tmpl("/tickets/{id}.json").set("id", id)), handleStatus()));
     }
 
+    public ListenableFuture<JobStatus<Ticket>> queueCreateTicketAsync(Ticket ticket) {
```

Comment:
@bsideup ptal, using the same terminology Zendesk use in their documentation.

Reply:
👍